### PR TITLE
Display limited availability message on events

### PR DESCRIPTION
**Pull request by davidrapson**
This is an amendment to PR https://github.com/guardian/membership-frontend/pull/368 after discussion with @davidmcdowell and Ben W.

**Dependant on PR https://github.com/guardian/membership-frontend/pull/373**

**Show limited availability messaging in header, like sold out or past events**
screen shot 2015-03-23 at 12 02 09

**Adjust position and messaging in sidebar**
screen shot 2015-03-23 at 12 02 19

### CODE REVIEW

File: frontend/app/model/Eventbrite.scala [hunk 1]
```diff
@@ -187,6 +189,10 @@ object Eventbrite {
       case t: InternalTicketing => t
     }
 
+    val limitedAvailabilityText = "Last tickets remaining"
```
Comment (feedmypixel) [inline frontend/app/model/Eventbrite.scala]: Potentially this text could sit in the fragment even though it is repeated. Would then be more explicit?

**Comment by feedmypixel** [comment]
needs a merge and a small comment bar that :+1:

File: frontend/app/model/Eventbrite.scala (hunk 1 above)
Comment (davidrapson) [inline frontend/app/model/Eventbrite.scala]: I think as it's repeated it feels nice to have it in here, I think we do the same for all other status messaging too.

**Comment by davidrapson** [comment]
@jamesoram Mind giving this a once over? I've set the limited availability number to ~150 so it will show on most events. You can change the following value if you want to confirm this in different scenarios https://github.com/guardian/membership-frontend/blob/display-n-tickets-left-new/frontend/app/model/Eventbrite.scala#L193

(This will be set to 15 once approved)

**Comment by jamesoram** [comment]
:+1:

**Pull request merged**